rust/src/dfa/jit.rs

```
@@ -0,0 +1,740 @@
+		let func: FuncId = self.module.declare_anonymous_function(&sig).unwrap();
+
+		self.context.func.signature = sig;
+
+		(Compilation {
+			module: &mut self.module,
+			asm: FunctionBuilder::new(&mut self.context.func, &mut self.function_context),
+			ptr_ty,
+		})
+		.compile(func, dfa);
+
+		now!(t0);
```

Clippy (1.96.0)

[error] cannot find macro `now` in this scope

(error)